### PR TITLE
Card: Adjusted styles to match pf ref. impl.

### PR DESCRIPTION
**Pull request by dtaylor113**
Only html/css changes to the header and footer areas to properly display the href and filter dropdown(s)

image

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
LGTM

**Comment by dtaylor113** [comment]
@erundle please review and merge if ok with changes -thanks!

File: src/card/basic/card-filter.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   {{currentFilter.label}}
   <span class="caret"></span>
 </button>
-<ul class="dropdown-menu">
+<ul class="dropdown-menu dropdown-menu-right" role="menu">
```
Comment (erundle) [inline src/card/basic/card-filter.html]: We should probably be using the angular-ui directive here.

File: src/card/basic/card.html [hunk 2]
```diff
@@ -1,42 +1,33 @@
...
-    <div class="card-pf-table">
-      <div class="card-pf-cell">
-        <h2 class="card-pf-title" ng-class="{'card-pf-filter-header': showFilterInHeader()}">{{headTitle}}</h2>
-      </div>
-      <div class="card-pf-cell text-right" ng-if="showFilterInHeader()">
-        <div class="dropdown btn-group">
-          <div ng-include="'card/basic/card-filter.html'"></div>
-        </div>
-      </div>
+  <div ng-if="showHeader()" ng-class="!showTitlesSeparator || showTitlesSeparator === 'true' ? 'card-pf-heading' : 'card-pf-heading-no-bottom'">
```
Comment (erundle) [inline src/card/basic/card.html]: Can this just be replaced with a function call?

File: src/card/basic/card-filter.html (hunk 1 above)
Comment (dtaylor113) [inline src/card/basic/card-filter.html]: I'm trying to use the exact html/css used in the base pf ref. impl.

File: src/card/basic/card.html (hunk 2 above)
Comment (dtaylor113) [inline src/card/basic/card.html]: Fixed

File: src/card/basic/card.directive.js [hunk 3]
```diff
@@ -180,9 +180,17 @@ angular.module('patternfly.card').directive('pfCard', function () {
         return ($scope.headTitle || $scope.showFilterInHeader());
       };
 
+      $scope.getHeaderClass = function () {
+        return (!$scope.showTitlesSeparator || $scope.showTitlesSeparator === 'true' ? 'card-pf-heading' : 'card-pf-heading-no-bottom');
+      };
```
Comment (waldenraines) [inline src/card/basic/card.directive.js]: Thoughts on pulling this into the html and using `ng-class`?

Comment (dtaylor113) [inline src/card/basic/card.directive.js]: That logic was in the html template, I moved into to the controller function per @erundle comment above :smile:  HTML is using ng-class

Comment (erundle) [inline src/card/basic/card.directive.js]: So I think Dave made this change in response to my code comment. What I was looking for was a scope variable that was just to to true or false so we weren't checking $scope.showTitlesSeparator === 'true' in the HTML. I am fine with the ng-class being in the HTML

Comment (erundle) [inline src/card/basic/card.directive.js]: See https://github.com/patternfly/angular-patternfly/blob/master/src/card/aggregate-status/aggregate-status-card.directive.js#L142

Comment (dtaylor113) [inline src/card/basic/card.directive.js]: Ok, changed to match that coding pattern

File: src/card/basic/card.html [hunk 4]
```diff
@@ -1,42 +1,33 @@
...
-      <div class="card-pf-cell">
-        <h2 class="card-pf-title" ng-class="{'card-pf-filter-header': showFilterInHeader()}">{{headTitle}}</h2>
-      </div>
-      <div class="card-pf-cell text-right" ng-if="showFilterInHeader()">
-        <div class="dropdown btn-group">
-          <div ng-include="'card/basic/card-filter.html'"></div>
-        </div>
-      </div>
+  <div ng-if="showHeader()" ng-class="{'card-pf-heading': shouldShowTitlesSeparator, 'card-pf-heading-no-bottom': !shouldShowTitlesSeparator}">
+    <div ng-if="showFilterInHeader()" class="dropdown card-pf-time-frame-filter">
```
Comment (erundle) [inline src/card/basic/card.html]: This might be easier to read
ng-class="shouldShowTilesSeparator ? 'card-pf-heading' | 'card-pf-heading-no-bottom'"

Comment (dtaylor113) [inline src/card/basic/card.html]: Fixed

Comment (erundle) [inline src/card/basic/card.html]: Thanks. @waldenraines any other issues or can I merge?

**Comment by waldenraines** [comment]
LGTM

**Pull request merged**